<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractionWidget</class>
 <widget class="QWidget" name="InteractionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="Interaction_Widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPlainTextEdit" name="breaksEdit"/>
        <widget class="QPlainTextEdit" name="inputEdit"/>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="text">
            <string>清除指令快</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateBreakButton">
           <property name="text">
            <string>更新断点</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateCodeButton">
           <property name="text">
            <string>上传代码</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
